dashboard/app/views/levels/_content.html.haml

```diff
@@ -4,8 +4,12 @@
 - app, data, content_class, source_level, hide_header, postcontent = %i(app data content_class source_level hide_header postcontent).map{ |x| local_assigns[x] }
 - level = source_level || @level
 
+- content_for(:head) do
+  %script{src: minifiable_asset_path('js/levels/_content.js')}
```

Comment:
`minifiable_asset_path` no longer exists! please use `webpack_asset_path` instead.

Reply:
...because we hash js assets via webpack instead of rails now. more context here: https://github.com/code-dot-org/code-dot-org/pull/30901 but really you can just use webpack_asset_path wherever you would previously have used `minifiable_asset_path` in the past.